Source/gg2/Objects/Menus/MenuController.events/Draw.xml

```diff
@@ -177,7 +177,24 @@ if(view_hview[0] &lt; 600)
                 val = "Right Mouse Button";
                 break;
             default:
-                val = chr(menu_get_var(i));
+                gotVar = menu_get_var(i);
```

Comment:
Make this a var and put it above the switch. Less duplication and less cluttering the instance with variables.